README.md

```diff
@@ -4,25 +4,23 @@ This repository contains wallpapers for use in rapps.
 
 You can download these wallpapers from the "ReactOS Standard Wallpapers" package in the ReactOS Applications Manager.
 
-| File Name                          | Author                                        | Origin                                                     |
-|------------------------------------|-----------------------------------------------|------------------------------------------------------------|
-| Angelus collection                 | Forum user "Angelus" (jedi.knight@wanadoo.es) | https://reactos.org/forum/viewtopic.php?p=50543#p50543     |
-| Angelus_02_ROSWP_4-3_1440x1080.jpg | See above                                     | See above                                                  |
-| Angelus_02_ROSWP_4-3_1024x768.jpg  | See above                                     | See above                                                  |
```

Comment:
You've included these two separate wallpapers within the "Angelus collection" ?

Reply:
Yes.